source/jormungandr/jormungandr/street_network/geovelo.py

```diff
@@ -104,8 +108,8 @@ def _make_request_arguments_bike_details(cls, bike_speed_mps):
     @classmethod
     def _make_request_arguments_isochrone(cls, origins, destinations, bike_speed_mps=3.33):
         return {
-            'starts': [cls._pt_object_summary_isochrone(o) for o in origins],
-            'ends': [cls._pt_object_summary_isochrone(o) for o in destinations],
+            'starts': (cls._pt_object_summary_isochrone(o) for o in origins),
```

Comment:
usjon has difficulties with generators as reported in several issues... what a pity...